app/metrics.yaml

```diff
@@ -1856,3 +1856,16 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+perf_measurements:
```

Comment:
nit: since this is really startup timelime, what about making the category `startup.timeline` and the metric `gecko_runtime_create`?

`perf_measurements` feels a bit generic given that there's other perf measurements being sent..